conf/services/customs-financials.cy.yml

```diff
@@ -0,0 +1,36 @@
+  - mae rhai PDFs yn cynnwys rhifau tudalennau anghywir yn y troedyn
+  - wrth ddewis y cysylltiad lawrlwytho, mae dogfennau’n lawrlwytho i’r ddyfais ond nid ydynt bob amser yn agor o fewn y porwr
+  - Bydd y gwasanaeth yn dod â sesiwn y defnyddiwr i ben heb roi’r dewis iddo o ymestyn y sesiwn.
+  - O ran y meysydd ar gyfer y cyfeiriad e-bost a’r cyfeiriad, efallai na chaiff y defnyddiwr yr opsiwn o lenwi’r meysydd hyn ar sail data a gadwyd yn y porwr yn flaenorol.
+  - Wrth newid i’r Gymraeg, nid yw’r cysylltiad syth ymlaen na’r cysylltiad allgofnodi wedi’u cyfieithu ac maent yn parhau i fod yn Saesneg.
+milestones:
+  - description: Nid oes gan y dogfennau PDF deitl. Mae hyn yn golygu nad yw diben y ddogfen yn cael ei gyflwyno i ddefnyddwyr darllenwyr sgrin, sy’n gorfod dibynnu ar enw’r ffeil ac mae hynny’n amwys. Nid yw hyn yn bodloni maen prawf llwyddiant 2.4.2 (Page Titled) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1.
+    date: 2021-08-31
+  - description: Nid yw’r dogfennau PDF wedi’u tagio, sy’n golygu nad yw eu cynnwys yn ddarllenadwy gan ddefnyddwyr darllenwyr sgrin. Nid yw hyn yn bodloni maen prawf llwyddiant 1.3.1 (Info and Relationships) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1.
+    date: 2021-08-31
+  - description: Mae’r PDFs yn cynnwys delweddau sydd heb destun amgen. Nid yw hyn yn bodloni maen prawf llwyddiant 1.1.1 (Non-text Content) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1.
+    date: 2021-08-31
+  - description: Mae dogfennau PDF yn cael eu lawrlwytho, ond nid ydynt yn agor mewn tab newydd. Ni roddir gwybod am yr ymddygiad hwn yn nhestun y cysylltiad lawrlwytho gweladwy. Nid yw hyn yn bodloni maen prawf llwyddiant 2.4.4 (Link Purpose In Context) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1.
+    date: 2021-08-31
+  - description: Pan mae’r sesiwn ar fin dod i ben, ni chaiff y defnyddiwr yr opsiwn o ymestyn ei sesiwn, ac nid oes rhybudd bod y sesiwn ar fin rhedeg allan o amser. Nid yw hyn yn bodloni maen prawf llwyddiant 2.2.1 (Timing Adjustable) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1
+    date: 2021-08-31
+  - description: O ran y meysydd mewnbwn ar gyfer y cyfeiriad a’r cyfeiriad e-bost, mae’r priodoleddau awtogwblhau ar goll, ac efallai y bydd hyn yn golygu na fydd y defnyddiwr yn gallu defnyddio’r data a gadwyd gan y porwr er mwyn llenwi’r meysydd hyn yn hawdd. Nid yw hyn yn bodloni maen prawf llwyddiant 1.3.5 (Identify Input Purpose) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1
+    date: 2021-08-31
+  - description: Pan gaiff y dudalen ei dangos yn Gymraeg, mae rhai cysylltiadau’n ymddangos yn Saesneg, ac nid oes ganddynt y briodoledd iaith berthnasol i ddangos eu bod mewn iaith wahanol. Nid yw hyn yn bodloni maen prawf llwyddiant 3.1.2 (Language of Parts) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1
+    date: 2021-08-31
+serviceLastTestedDate: 2021-04-12
+statementVisibility: public
+statementCreatedDate: 2019-09-16
+statementLastUpdatedDate: 2021-04-13
```

Comment:
This date should be updated to `2021-05-04`

Reply:
fixed